nbstripout.py

```diff
@@ -336,13 +376,18 @@ def main():
         print(__version__)
         sys.exit(0)
 
+    try:
+        extra_keys = check_output(['git', 'config', 'filter.nbstripout.extrakeys']).strip()
```

Comment:
This will only fail if `git` is not in the path / CWD is not inside a git repository i.e. when using `nbstripout` in standalone mode, correct?

Reply:
No, actually this command will run even outside a repo (defaulting to the global config).
It will however exit with non-zero status if the config `filter.nbstripout.extrakeys` is not found/defined.

It's possible you might prefer going for a `$HOME/.nbsripoutrc` file or similar, but it seemed neater using `git`'s functionality.

Reply:
In that case, how about using
```suggestion
        extra_keys = getoutput('git config filter.nbstripout.extrakeys').strip()
```

Reply:
would have to define `getoutput`

Reply:
Sorry, I mean `subprocess.getoutput` instead of `subprocess.check_output` (`getoutput` doesn't throw when the command exits with non-0 status)

Reply:
that doesn't exist (python2)

Reply:
True, let's leave as is then